content/blog/improved-preview-experience/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Improved Pulumi Previews"
+date: 2020-10-27
+draft: false
+meta_desc: "Announcing a significant improvement to the pulumi preview experience."
+#meta_image: 
+authors:
+    - paul-stack
+tags:
+    - enhancement
+    - pulumi
+---
+
+Today we are announcing a minor but significant improvement to the Pulumi [preview]({{< relref "/docs/reference/cli/pulumi_preview" >}}
+experience.
+
+<!--more-->
+
+## Change in Behaviour
+
```

Comment:
It might be good to have a quick 1-2 sentences explaining the previous behavior before providing the example.

Reply:
There's an example of this, but it might be good to call it out via the header.

# Previous behaviour

# New Behaviour